hack/manifest-gen/crd_patches/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - all-crds.yaml
+commonLabels:
+  helm.sh/chart: '{{ include "eck-crds.chart" . }}'
+  app.kubernetes.io/version: '{{ .Chart.AppVersion }}'
+  app.kubernetes.io/managed-by: '{{ .Release.Service }}'
+  app.kubernetes.io/name: '{{ include "eck-crds.name" . }}'
+  app.kubernetes.io/instance: '{{ .Release.Name }}'
```

Comment:
No, these labels only make sense when a tool like Helm is managing them. If someone installs the operator using `all-in-one.yaml`, then there are no sensible values that can be used with them.

Reply:
If those labels only makes sense in the helm chart context, why do they appear in `hack/manifest-gen` and not in `deploy/`?

Reply:
Because `manifest-gen.sh` is updating the Helm chart with the latest CRDs.  Helm needs all resources it deploys to have those labels. Since controller-gen doesn't add any labels to the CRDs it generates, we have to take them, apply a kustomize script to add the labels and then copy them over to the subchart.